src/utils/StreamHighlightUtils.ts

```diff
@@ -136,6 +137,17 @@ export function getRestHighlightsForAllTerms(
     .value();
 }
 
+function getUniqueTerms(termsToHighlight, phrasesToHighlight): string[] {
+  const sortedTerms = _.keys(termsToHighlight).sort(termsSorting);
+  const phrasesTerms = _.chain(phrasesToHighlight)
```

Comment:
Ok, I saw your response and you have a fair point. I would just ask to adjust the spelling to `phraseTerms` please.

Reply:
Do you think `termsFromPhrases` would be a better name? Since they come from multiple phrases